Hoon/AndroidStudy_Hoon/app/src/main/java/com/study/androidstudy_hoon/MainActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.study.androidstudy_hoon
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import com.study.androidstudy_hoon.databinding.ActivityMainBinding
+import com.study.androidstudy_hoon.presenter.main.MainViewModel
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var viewModel: MainViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
```

Comment:
ViewModel을 ktx로 선언했으면 아래와 같이 별도의 초기화 없이 수정 가능합니다~
```suggestion
setContentView(binding.root)
```

Reply:
수정완료했습니다 👍